### PR TITLE
reverting pointsize change

### PR DESCRIPTION
**Pull request by ldanzinger**
@DonMoJa Please review. Reverting the point size changes. They had more negative impacts than positive.

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/EditData/EditFeatureAttachments/EditFeatureAttachments.qml [hunk 1]
```diff
@@ -93,11 +93,7 @@ EditFeatureAttachmentsSample {
                     margins: 10
                 }
 
-                text: "Attachments"
-                font {
-                    bold: true
-                    pointSize: 20
-                }
+                text: "Attachments"; font {bold: true; pixelSize: 20;}
```
Comment (DonMoJa) [inline ArcGISRuntimeSDKQt_CppSamples/EditData/EditFeatureAttachments/EditFeatureAttachments.qml]: Formatting seems a bit odd in a few places. Do we want to revert back to these or no?

File: ArcGISRuntimeSDKQt_CppSamples/EditData/EditFeatureAttachments/EditFeatureAttachments.qml [hunk 2]
```diff
@@ -108,12 +104,7 @@ EditFeatureAttachmentsSample {
                 }
                 spacing: 15
                 Text {
-                    text: "+"
-                    font {
-                        bold: true
-                        pointSize: 40
-                    }
-                    color: "green"
```
Comment (DonMoJa) [inline ArcGISRuntimeSDKQt_CppSamples/EditData/EditFeatureAttachments/EditFeatureAttachments.qml]: same

File: ArcGISRuntimeSDKQt_CppSamples/EditData/EditFeatureAttachments/EditFeatureAttachments.qml [hunk 3]
```diff
@@ -108,12 +104,7 @@ EditFeatureAttachmentsSample {
...
+                    text: "+"; font {bold: true; pixelSize: 40;} color: "green"
```
Comment (DonMoJa) [inline ArcGISRuntimeSDKQt_CppSamples/EditData/EditFeatureAttachments/EditFeatureAttachments.qml]: same

File: ArcGISRuntimeSDKQt_CppSamples/EditData/EditFeatureAttachments/EditFeatureAttachments.qml [hunk 4]
```diff
@@ -124,12 +115,7 @@ EditFeatureAttachmentsSample {
                     }
                 }
                 Text {
-                    text: "-"
-                    font {
-                        bold: true
-                        pointSize: 40
-                    }
-                    color: "red"
```
Comment (DonMoJa) [inline ArcGISRuntimeSDKQt_CppSamples/EditData/EditFeatureAttachments/EditFeatureAttachments.qml]: same

File: ArcGISRuntimeSDKQt_QMLSamples/EditData/EditFeatureAttachments/EditFeatureAttachments.qml [hunk 5]
```diff
@@ -172,11 +172,7 @@ Rectangle {
                     margins: 10
                 }
 
-                text: "Attachments"
-                font {
-                    bold: true
-                    pointSize: 20
-                }
+                text: "Attachments"; font {bold: true; pixelSize: 20;}
```
Comment (DonMoJa) [inline ArcGISRuntimeSDKQt_QMLSamples/EditData/EditFeatureAttachments/EditFeatureAttachments.qml]: same

File: ArcGISRuntimeSDKQt_QMLSamples/EditData/EditFeatureAttachments/EditFeatureAttachments.qml [hunk 6]
```diff
@@ -187,12 +183,7 @@ Rectangle {
                 }
                 spacing: 15
                 Text {
-                    text: "+"
-                    font {
-                        bold: true
-                        pointSize: 40
-                    }
-                    color: "green"
```
Comment (DonMoJa) [inline ArcGISRuntimeSDKQt_QMLSamples/EditData/EditFeatureAttachments/EditFeatureAttachments.qml]: same

File: ArcGISRuntimeSDKQt_QMLSamples/EditData/EditFeatureAttachments/EditFeatureAttachments.qml [hunk 7]
```diff
@@ -187,12 +183,7 @@ Rectangle {
...
+                    text: "+"; font {bold: true; pixelSize: 40;} color: "green"
```
Comment (DonMoJa) [inline ArcGISRuntimeSDKQt_QMLSamples/EditData/EditFeatureAttachments/EditFeatureAttachments.qml]: same

File: ArcGISRuntimeSDKQt_QMLSamples/EditData/EditFeatureAttachments/EditFeatureAttachments.qml [hunk 8]
```diff
@@ -203,12 +194,7 @@ Rectangle {
                     }
                 }
                 Text {
-                    text: "-"
-                    font {
-                        bold: true
-                        pointSize: 40
-                    }
-                    color: "red"
+                    text: "-"; font {bold: true; pixelSize: 40;} color: "red"
```
Comment (DonMoJa) [inline ArcGISRuntimeSDKQt_QMLSamples/EditData/EditFeatureAttachments/EditFeatureAttachments.qml]: same

File: ArcGISRuntimeSDKQt_CppSamples/EditData/EditFeatureAttachments/EditFeatureAttachments.qml [hunk 9]
```diff
@@ -124,12 +115,7 @@ EditFeatureAttachmentsSample {
...
+                    text: "-"; font {bold: true; pixelSize: 40;} color: "red"
```
Comment (DonMoJa) [inline ArcGISRuntimeSDKQt_CppSamples/EditData/EditFeatureAttachments/EditFeatureAttachments.qml]: same

**Comment by ldanzinger** [comment]
thanks @DonMoJa. Agreed, probably good to just clean this up now while at it. Updated

File: ArcGISRuntimeSDKQt_CppSamples/EditData/EditFeatureAttachments/EditFeatureAttachments.qml [hunk 10]
```diff
@@ -93,7 +93,11 @@ EditFeatureAttachmentsSample {
                     margins: 10
                 }
 
-                text: "Attachments"; font {bold: true; pixelSize: 20;}
+                text: "Attachments"
+				font {
+				    bold: true
+					pixelSize: 20
+                }
```
Comment (DonMoJa) [inline ArcGISRuntimeSDKQt_CppSamples/EditData/EditFeatureAttachments/EditFeatureAttachments.qml]: Looks like this one slipped through.

Comment (ldanzinger) [reply ArcGISRuntimeSDKQt_CppSamples/EditData/EditFeatureAttachments/EditFeatureAttachments.qml]: updated @DonMoJa

**Pull request merged**